docs-src/cloud/migration.md

```diff
@@ -0,0 +1,88 @@
+
+If you are performing a live migration, make sure your Worker capacity can support the migration load.
+Both a [Signal](/concepts/what-is-a-signal) and a [Query](/concepts/what-is-a-query) will be executed during the course of the migration.
+Also, the Query API loads the entire history of Workflows into Workers to compute the result (if they are not already cached).
+That means that your self-hosted cluster Worker capacity will need to support having those executions in memory to serve those requests.
+The volume of these requests might be quite high to execute against all the matches to a `ListFilter`.
+
+### Considerations when resuming Workflows on a new Cluster or Namespace
+
+- **Skipping Steps:** If your Workflow steps cannot guarantee idempotency, determine if you need to skip those steps when resuming the execution in the target Namespace.
+- **Elapsed Time:** If your Workflow is “resuming sleep” when in the target Namespace, determine how you will calculate the delta for the sleep invocation in the new execution.
+- **Child Relationships:** If your Workflow has Child Workflow relationships (other than Detached Parent Close Policy children), determine how you can pass the state of those children into the parent to execute the child in a resumed state.
+- **Heartbeat state:** If you have long running activities relying on heartbeat state, determine how you can resume these activities in the target Namespace.
+- If Child Workflows are the same type as their Parent types, those will be returned in List Filters being used to gather relevant executions. Unless these are Detached `ParentClosePolicy` children, this is not what you want since the Parent/Child relationship will not be carried over to the target Namespace.
+- Long running activities that use heartbeat details will not receive the latest details in the target Namespace.
+- Duration between Awaitables inside Workflow definition needs to be considered for elapsed time accuracy when resuming in the target Namespace.
+- Signaling directly from one Workflow to another needs to handle NotFound executions in the target Namespace since they may resume out of order.
+
+### Other considerations when migrating
+
+- Have you added an mTLS certificate to your Temporal Namespace? Review our [documentation for adding a certificate to your Temporal Cloud account](/cloud/certificates-intro) for more information.
+- There are differences in how metrics are generated in self-hosted Temporal and Temporal Cloud. Review the [documentation on Temporal Cloud metrics](/cloud/metrics-intro) for more information.
+- Consider the implications for [security and access to your cluster](/cloud/security-cloud-intro).
+- Review your current load (actions per second) and speak to your Account Executive and Solutions Architect so we can set appropriate Namespace [limits](/cloud/limits).
```

Comment:
I'd include Namespace in the link text for /cloud/limits. Potential rewording: "We can help adjust your [Namespace limits](/cloud/limits) to better meet your current actions-per-second load in consultation with your Account Executive and Solutions Architect."